app/views/users/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_for user do |f| %>
```

Comment:
"Rails will also automatically set the class and id of the form appropriately: a form creating an article would have id and class new_article. If you were editing the article with id 23, the class would be set to edit_article and the id to edit_article_23. These attributes will be omitted for brevity in the rest of this guide."
form hiểu được lúc nào post lúc nào put là do rails chỉ cho ạ